src/humam/data_preprocessing/__init__.py

```diff
@@ -0,0 +1 @@
+from . import connectivity, cytoarchitecture
```

Comment:
Check if this could better be
```suggestion
__all__ = ["connectivity", "cytoarchitecture"]
```